src/main/java/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,19 @@
+package converter;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> converterToTransaction(List<String> data) {
+        return data.stream()
+                .skip(1) // Skip header
+                .map(line -> {
+                    String[] parts = line.split(",");
```

Comment:
Consider checking if the `parts` array has the expected number of elements before accessing them to avoid `ArrayIndexOutOfBoundsException`.